_sass/components/_accordions.scss

```diff
@@ -21,16 +21,36 @@
     .accordion-button:after {
       content: "\f077";
     }
-
-    .accordion-content {
+    .accordion-button_more {
+      display: none;
+    }
+    .accordion-button_less {
+      &::after {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets